### PR TITLE
Updates mansus grasp ability icon and heretic background sprites

### PR DESCRIPTION
**Pull request by Djiq**
## About The Pull Request
Changes how mansus grasp and heretic background looks

image
image

## Why It's Good For The Game
When i made those two sprites I was like 4 years younger, a lot worse at spriting, and possibly high and/or drunk. Needless to say they are shit, so i updated them.

## Changelog
:cl:
image: Changes mansus grasp ability icon and heretic background sprites
/:cl:

### CODE REVIEW

**Comment by SmArtKar** [comment]
How do they look with abilities? I feel like the background is a huge downgrade from what we have right now, doubt that it'll look very good with abilities on it

**Comment by Djiq** [comment]
> How do they look with abilities? I feel like the background is a huge downgrade from what we have right now, doubt that it'll look very good with abilities on it

the curent icon is literally the cult icon hue-shifted green with an eye shoddily drawn on it (i was drunk)

**Comment by Metekillot** [comment]
Maybe a little bit too blasphemous? The art is stellar and I'm personally not churchgoing these days, but I felt a little stir of HOW DARE YOU in my Raised Catholic Loins seeing The Creation of Adam used as a template for a baddie who carves people open on a circle

**Comment by Djiq** [comment]
> Maybe a little bit too blasphemous? The art is stellar and I'm personally not churchgoing these days, but I felt a little stir of HOW DARE YOU in my Raised Catholic Loins seeing The Creation of Adam used as a template for a baddie who carves people open on a circle
> 
> I'm not spriters so take it with a grain of salt but maybe bloodying the hand and making it look trippy and fucked up so we're not wagging the dog so hard at Christianity?

i mean yeah but the parallel of a god literally reaching and impacting real world is a powerful image

**Comment by Pumpkinoe** [comment]
i think having a realistic hand in one of your hand slots is kind of funny but also super duper unfitting for the game's art style and how communicative it is as an ability

**Comment by Djiq** [comment]
> i think having a realistic hand in one of your hand slots is kind of funny but also super duper unfitting for the game's art style and how communicative it is as an ability

this is the ability icon not the inhand sprite

**Comment by Pumpkinoe** [comment]
> > i think having a realistic hand in one of your hand slots is kind of funny but also super duper unfitting for the game's art style and how communicative it is as an ability
> 
> this is the ability icon not the inhand sprite

oh! then fair enough

**Comment by Kiromaka** [comment]
Ye, the hand should be bloody, or at least darker in colours. Doenst really fit the - bloody scary cultist that eats human flesh for recreation.

**Comment by DATA-xPUNGED** [comment]
I REALLY dig the new background, not so much the hand, I vibe with what you're trying to do with it but I don't think it really fits the grasp

**Comment by Djiq** [comment]
> I REALLY dig the new background, not so much the hand, I vibe with what you're trying to do with it but I don't think it really fits the grasp

this is a part of the slight retheming im doing of heretics, it will be more thematic once i get to rewording a few abilities

**Comment by Kiromaka** [comment]
> 

Just go full cultist simulator.

**Comment by DATA-xPUNGED** [comment]
> > I REALLY dig the new background, not so much the hand, I vibe with what you're trying to do with it but I don't think it really fits the grasp
> 
> this is a part of the slight retheming im doing of heretics, it will be more thematic once i get to rewording a few abilities

YOu know what? I dig that very hard, I will trust your judgement

**Review by Wallemations — CHANGES_REQUESTED** [review]
I'm not 100% familiar with how heretics work since I've never rolled them, so would you mind showing an in-game screenshot of how these look? I'm assuming since you said background that one of those shows up behind other icons in the HUD?

**Comment by Djiq** [comment]
> I'm not 100% familiar with how heretics work since I've never rolled them, so would you mind showing an in-game screenshot of how these look? I'm assuming since you said background that one of those shows up behind other icons in the HUD?

image

**Comment by Wallemations** [comment]
> > I'm not 100% familiar with how heretics work since I've never rolled them, so would you mind showing an in-game screenshot of how these look? I'm assuming since you said background that one of those shows up behind other icons in the HUD?
> 
> 
> 
> image
> 
> 

Ok I'll be entirely honest
The eyeball's current sprite makes it pretty hard to parse some of the icons like the heart one & the fireball. I'm sure this could be fixed with a simple opacity change.

As for the hand, I'm wrestling in my head whether or not it works. As it currently stands with the rest of the icons, it feels very out of place. I think it would take a whole respriting of all the icons like the changeling icons had in order to make it really fit in tbh. However I'm not about to tell you "go resprite every other icon", which is why I'm struggling on whether or not it fits.

**Comment by Djiq** [comment]
> > > I'm not 100% familiar with how heretics work since I've never rolled them, so would you mind showing an in-game screenshot of how these look? I'm assuming since you said background that one of those shows up behind other icons in the HUD?
> > 
> > 
> > image
> 
> Ok I'll be entirely honest The eyeball's current sprite makes it pretty hard to parse some of the icons like the heart one & the fireball. I'm sure this could be fixed with a simple opacity change.
> 
> As for the hand, I'm wrestling in my head whether or not it works. As it currently stands with the rest of the icons, it feels very out of place. I think it would take a whole respriting of all the icons like the changeling icons had in order to make it really fit in tbh. However I'm not about to tell you "go resprite every other icon", which is why I'm struggling on whether or not it fits.

i can remove the mansus hand from this pr and re-pr it after i do some more work on it along with a thematic rewrite, as it stands this was supposed to be a part 1 of my slight retheming of heretics and i understand that i may not fit right *now*, i will lower the opacity on the eye thing in the background sprite and update the pictures

**Comment by Xackii** [comment]
hahaha eye funny eye hahahhaa i like this eye. You can add this backgound for void only(or flash but more like void) because horror vibes + void looks on you+ immersive

**Comment by Wallemations** [comment]
> i will lower the opacity on the eye thing in the background sprite and update the pictures

Been waiting on this, just making sure you don't forget!
